src/System.Drawing.Design.Primitives/src/System/Drawing/Design/ToolboxComponentsCreatingEventArgs.cs

```diff
@@ -0,0 +1,37 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.ComponentModel.Design;
+namespace System.Drawing.Design
+{
+    /// <summary>
+    /// Provides data for the 'ToolboxComponentsCreatingEventArgs' event that occurs
+    /// when components are added to the toolbox.
+    /// </summary>
+    public class ToolboxComponentsCreatingEventArgs : EventArgs
+    {
+        private readonly IDesignerHost host;
```

Comment:
Nit: prefix with an underscore, unless the name must remain like this for some reason, in which case a comment should explain why so we don't accidentally change it in the future